### DISCUSSION TITLE
orcaslicer and windows 11 PRO 24H2

### DISCUSSION THREAD

**Discussion by alf26540** [OrcaSlicer/OrcaSlicer#7255] [General]
hello to all 
I am experiencing a problem on a Windows 11 PRO PC. to install OrcaSlicer I am obliged to force the protections of Windows 11. At most this does not prevent its installation. Because I can fly my ELEGOO NEPTUNE 4Max. However, since I start the cutting of a piece, I have a crash of Windows and a restart of the PC . 
Is there a particular configuration of windows to do where a problem in the installation of OrcaSlicer since I force the installation 

thanks for your answer

**Comment by nekrosoft13** [comment]
I don't fully understand what you mean...

"However, since I start the cutting of a piece, I have a crash of Windows and a restart of the PC" This could be a video card/display issue. 
By cutting, do you mean slicing? 

I haven been using Orca with Windows 11 24H2 since around May of 2024, and not experienced any crashes.
